bin/gobblin.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+classpath() {
+  DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+  echo "$DIR/../lib/*";
+}
+
+for i in "$@"
+do
+  case "$1" in
+    "classpath")
+      classpath
+      exit
+  esac
+done
+
+CLASSPATH=$(classpath)
+
+java -cp "$CLASSPATH" $GOBBLIN_OPTS gobblin.runtime.cli.GobblinCli $@
```

Comment:
JVM args are specified with GOBBLIN_OPTS (this is the way Hadoop does it as well). work dir and log dir can be set already using something like `-configuration gobblin.wor.dir:/my/work/dir`. This is intended as a basic, first iteration of the script, but I can add this functionality now if you think it is important.